_components/search-bar.md

```diff
@@ -8,55 +8,40 @@ title: Search Bar
 
   <div class="usa-grid">
     <div class="usa-width-one-half">
-      <form class="usa-search usa-search-big">           
-        <fieldset>
```

Comment:
Why did we remove fieldsets and legends? Did you check with @nickbristow about this?